core/src/main/java/org/apache/calcite/jdbc/CalciteSchema.java

```diff
@@ -220,6 +223,16 @@ public FunctionEntry add(String name, Function function) {
     return entry;
   }
 
+  public FunctionEntry add(String name, Macro macro) {
```

Comment:
Can multiple macros with the same name but different number of parameters be defined?

Reply:
They can't as far as I know.

Reply:
Ok sounds good.